src/vivarium_testing_utils/automated_validation/interface.py

```diff
@@ -3,20 +3,26 @@
 from pathlib import Path
 
 import pandas as pd
+import vivarium_inputs as vi
```

Comment:
This still needs to be locally imported where it's called to enable building an environment without vivarium inputs. I believe the same is true for the ArtifactException?